pkg/redirectpolicy/manager.go

```diff
@@ -79,16 +96,30 @@ type Manager struct {
 	policyPods map[podID][]policyID
 	// Stores redirect policy configs indexed by policyID
 	policyConfigs map[policyID]*LRPConfig
+	// Stores mapping of pod endpoints to redirect policies that select the pods
+	policyEndpoints map[podID]sets.Set[policyID]
+
+	noNetnsCookieSupport bool
 }
 
-func NewRedirectPolicyManager(svc svcManager, lpr agentK8s.LocalPodResource) *Manager {
+func NewRedirectPolicyManager(svc svcManager, lpr agentK8s.LocalPodResource, epM endpointManager) *Manager {
+	skipLBMap, err := lbmap.NewSkipLBMap()
+	if err != nil {
+		log.Warnf("failed to init cilium_skip_lb maps: %v"+
+			"Check the local redirect policy documentation if you are using"+
+			"CiliumLocalRedirectPolicy where it's required to"+
+			"skip load-balancing in certain cases", err)
```

Comment:
I understand your point, but it seems like an overkill as not all users would want to use the spec flag. 
But I'll update the warning to make it more user friendly.

Reply:
Where I'm coming from here is, if I'm a user who is relying on this field, I'd be compelled to check for this warning log every time I restart a Cilium Agent because there's no other indication given by Cilium that the field is working or not.

Reply:
I ended up refactoring this a bit to lazy initialize the maps only when needed. So if a policy that needs this functionality is not applied because of errors around the skip_lb maps, users can check the logs as expected.